datasets/seq_tinyimagenet.py

```diff
@@ -15,8 +15,7 @@
 from torch.utils.data import Dataset
 
 from datasets.transforms.denormalization import DeNormalize
-from datasets.utils.continual_dataset import (ContinualDataset,
-                                              store_masked_loaders)
+from datasets.utils.continual_dataset import ContinualDataset
 from datasets.utils.validation import get_train_val
 from utils.conf import base_path_dataset as base_path
 
```

Comment:
Line 144: `store_masked_loaders( )` function is not defined

Reply:
removed in the final code version on the branch